app/helpers/dashboard/associated_users_helper.rb

```diff
@@ -19,29 +19,25 @@
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 module Dashboard::AssociatedUsersHelper
-  GLYPH_EDIT = 'glyphicon glyphicon-edit'.freeze()
-  GLYPH_REMOVE = 'glyphicon glyphicon-remove'.freeze()
-  BTN_WARNING = 'btn btn-warning'.freeze()
-  BTN_DANGER = 'btn btn-danger'.freeze()
-
+  
```

Comment:
If we're going to do this, it needs to be universally done in the Application Helper